3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -1,35 +1,55 @@
-'use strict';
-/*------------------------------------------------------------------------------
-Complete the four functions provided in the starter `index.js` file:
-
-`fetchData`: In the `fetchData` function, make use of `fetch` and its Promise 
-  syntax in order to get the data from the public API. Errors (HTTP or network 
-  errors) should be logged to the console.
-
-`fetchAndPopulatePokemons`: Use `fetchData()` to load the pokemon data from the 
-  public API and populates the `<select>` element in the DOM.
-  
-`fetchImage`: Use `fetchData()` to fetch the selected image and update the 
-  `<img>` element in the DOM.
-
-`main`: The `main` function orchestrates the other functions. The `main` 
-  function should be executed when the window has finished loading.
-
-Try and avoid using global variables. Instead, use function parameters and 
-return values to pass data back and forth.
-------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
+const buttonGetPokemon = document.createElement('button');
+document.body.appendChild(buttonGetPokemon);
+
+buttonGetPokemon.setAttribute('type', 'submit');
```

Comment:
A button with `type="submit"` only makes sense inside a form. (And in that case, you should not attach a click handler to the button, but a submit handler to the form.) In this situation, you're just attaching a click handler, which is fine, and then you don't need `type="submit"`.